geospaas/base_viewer/views.py

```diff
@@ -62,6 +63,8 @@ def set_context(cls, form, page_obj):
         context = {}
         context['form'] = form
         context['page_obj'] = page_obj
+        context['show_local'] = settings.SHOW_LOCAL_ADDRESS
```

Comment:
`context['show_local_address']` would be clearer

Reply:
Done in the next commit.